libcontainer/cgroups/devices/devices_emulator.go

```diff
@@ -1,5 +1,23 @@
 // +build linux
 
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * Copyright (C) 2020 Aleksa Sarai <cyphar@cyphar.com>
+ * Copyright (C) 2020 SUSE LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Since the whole project is licensed under Apache 2.0, perhaps the below is redundant (iow having SPDX-License-Identifier and the copyright statement is sufficient)?

Reply:
I take it back; https://spdx.dev/ids/ says the standard boilerplate is also needed